### ISSUE TITLE
Bad Authentication Data when trying to get request token

### ISSUE THREAD

**Issue by siddharthgopi** [reneraab/node-twitter-api#43]
var twitterAPI = require('node-twitter-api');
	var twitter = new twitterAPI({
	  consumer_key: "removed",
	  consumer_secret: "removed",
	  callback: 'http://52.43.53.156:3000'
	});

**response:**

Error getting OAuth request token : [object Object]
statusCode=400
data={"errors":[{"code":215,"message":"Bad Authentication data."}]}

--

My callback in this code is the same as what i entered in the twitter web interface.

**Comment by siddharthgopi** [comment]
Is it correct that I have to take consumer_key and consumer_secret without any further modification directly from the twitter dashboard? as in the api_key and api_secret in https://apps.twitter.com/app/12851437/keys

None of the libraries are working in fact. I also tried changing the twitter api version to 1.1. No difference.

**Comment by reneraab** [comment]
Yes, you need to use the API key and API secret. What exactly have you been trying to do, when you received the error?

**Comment by siddharthgopi** [comment]
Hi Rene, i sent you a mail at mail@reneraab.org

**Comment by siddharthgopi** [comment]
I literally haven't made one inch of a progress in the last 10 hours
